src/database/DataTypes/RequisitionItem.js

```diff
@@ -30,8 +30,9 @@ export class RequisitionItem extends Realm.Object {
   }
 
   get suggestedQuantity() {
-    const daysToSupply = this.requisition ? this.requisition.daysToSupply : 0;
-    return Math.ceil(Math.max(this.dailyUsage * daysToSupply - this.stockOnHand, 0));
+    const { daysToSupply } = this.requisition || { daysToSupply: 0 };
```

Comment:
This doesn't look like eslint change. (pretty smart though, but previous code is more readable)

Reply:
Reverted.